src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js

```diff
@@ -108,6 +108,7 @@ class ChargeFeeFine extends React.Component {
     const { intl: { formatMessage } } = this.props;
     const item = (selectedLoan.id) ? selectedLoan.item : this.item;
 
+    this.type = type;
```

Comment:
why do we need it here?

Reply:
The code location does not have any special meaning

Reply:
I see several type assigning in this file, not sure that it is correct

Reply:
Yes, type assigning is implemented in two methods for different flows

Reply:
We we can't just pass notify property to a local variable and to newAction method instead of store it in class property?

Reply:
Because we anyway should save the `notify` value from this method. I will store `notify` value in state then.